tests/repl/test_engine.rs

```diff
@@ -75,6 +75,12 @@ fn in_used_twice_and_also_in_pipeline() -> TestResult {
     )
 }
 
+// #13441
+#[test]
+fn in_used_in_range() -> TestResult {
+    run_test(r#"6 | 3..$in | math sum"#, "18")
```

Comment:
I think we can add one more test for `6 | $in..10 | math sum`

Reply:
sure, why not